zerver/lib/email_mirror.py

```diff
@@ -72,6 +79,7 @@ def get_missed_message_token_from_address(address):
     return msg_string[2:]
 
 def create_missed_message_address(user_profile, message):
+    # type: (UserProfile, UserMessage) -> str
```

Comment:
I think message here is a Zulip `Message` from zerver.models (not a UserMessage)